23 спринт

KateSilencio opened the pull request:
Версия 1

KateSilencio committed:
спринт 23
прототип
сломанная версия

KateSilencio committed:
спринт 23
прототип
сломанная версия

KateSilencio committed:
спринт 23
прототип
сломанная версия

KateSilencio committed:
спринт 23
прототип
сломанная версия

KateSilencio committed:
спринт 23
прототип

KateSilencio committed:
спринт 23
Версия 1

anastasianadelshtehel commented on app/src/main/java/com/example/playlistmaker/medialib/ui/EditPlaylistFragment.kt:
```diff
@@ -0,0 +1,77 @@
+        binding.inputTextTitle.setText(args.playlist.title)
+        binding.inputTextDescription.setText(args.playlist.description ?: "")
+        args.playlist.coverUri?.let { uri ->
+            selectedImageUri = uri.toUri()
```
Преобразование String в Uri (toUri()) происходит в нескольких местах. Лучше сделать это один раз при инициализации.

anastasianadelshtehel commented on app/src/main/java/com/example/playlistmaker/medialib/ui/EditPlaylistFragment.kt:
```diff
@@ -0,0 +1,77 @@
+        // Заполняем данные
+        binding.inputTextTitle.setText(args.playlist.title)
+        binding.inputTextDescription.setText(args.playlist.description ?: "")
+        args.playlist.coverUri?.let { uri ->
```
Можно объединить логику:
```
args.playlist.coverUri?.let { uri ->
    selectedImageUri = uri.toUri()
    loadExistingCover(uri)
}
```

KateSilencio committed:
спринт 23
Версия 2
Изменения:
добавлен новый View в fragment_playlist_screen.xml
В EditPlaylistFragment добавлена обработка системной кнопки Назад
